magprime/configspec.ini

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
@binary1230: This is the first time we're actually putting configuration in the plugin's config files.  Usually we just add sub-sections to the main repo, since config sections such as ``[enums]`` are specifically designed to support that.

In this case, we have a config section which needs its own spec, and it doesn't belong in the core repo, hence we've put it here.  That's fine, except that now puppet will need to know how to write to ``plugins/magprime/development.ini`` in addition to ``plugins/uber/deveopment.ini``.

In the short run we can just edit those files by hand, which is probably better than waiting too long to send out the season pass emails.